change/@azure-msal-browser-30eb62a8-70f1-4fc5-b9d1-12fd3447960e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
AFAIK this doesn't change the API surface, right?

Reply:
Right, this is all internal implementation details. Made it minor just for the sheer volume of changes, acknowledging the small risk that something might break as a result